gratipay/models/package/team.py

```diff
@@ -0,0 +1,79 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import uuid
+
+from gratipay.models.team import Team as _Team
+
+
+class Team(object):
+    """A :py:class:`~gratipay.models.package.Package` can have a
+    :py:class:`~gratipay.models.team.Team` associated with it.
+    """
+
+    @property
+    def team(self):
+        """A computed attribute, the :py:class:`~gratipay.models.team.Team`
+        linked to this package if there is one, otherwise ``None``. Makes a
+        database call.
+        """
+        return self.load_team(self.db)
+
+
+    def load_team(self, cursor):
+        """Given a database cursor, return a
+        :py:class:`~gratipay.models.team.Team` if there is one linked to this
+        package, or ``None`` if not.
+        """
+        return cursor.one( 'SELECT t.*::teams FROM teams t WHERE t.id='
+                           '(SELECT team_id FROM teams_to_packages tp WHERE tp.package_id=%s)'
+                         , (self.id,)
```

Comment:
Might be my recent lack of exposure to Python mixin classes, but this actually feels a bit backward. You mix it in with Package, but instead of Package leveraging some of the functionality of the mixed-in Team, Team leverages things like `self.id` that must be in Package.